tool/README.md

```diff
@@ -19,87 +19,41 @@ Make sure:
 
 ### Prepare the release
 
-#### Create a branch for your release.
-
-```shell
-cd ~/path/to/devtools
-```
-
-```shell
-git checkout master && \
-git pull upstream master && \
-git checkout -b release_$(date +%s);
-```
 
 #### Update the DevTools version number
 
-Run the `tool/update_version.dart` script to update the DevTools version.
-- For regular monthly releases, use `minor`:
-   ```shell
-   dart tool/update_version.dart auto --type minor
-   ```
-- To manually set the version:
-   ```shell
-   dart tool/update_version.dart manual --new-version 1.2.3
-   ```
-- To automatically update the version by `major`, `minor`, `patch`, or `dev`:
-   ```shell
-   dart tool/update_version.dart auto --type patch
-   ```
-- Pre-release versions can be stripped with:
-   ```shell
-   dart tool/update_version.dart auto --type release
-   ```
-
-Verify:
-* that this script updated the pubspecs under packages/
-* updated all references to those packages.
-*  make sure that the version constant in `packages/devtools_app/lib/devtools.dart` was updated
+- Make sure your working branch is clean
+- Run the `tool/release_helper.sh` script with `minor` or `major`.
+   `./tool/release_helper.sh [minor|major]`
+- This creates 2 branches for you:
+    - Clean Branch
+    - Next Branch
+- You will need to prepare and submit both of these branches, when you are doing a full release.
```

Comment:
'you will need' does not sound as instruction

How about: If you are doing a full release, prepare and submit...